src/services/staking_data_service.ts

```diff
@@ -164,6 +178,19 @@ const nextEpochStatsQuery = `
                     AS operator_share
             FROM staking.operator_share_changes
         )
+        , past_7d_fills_fills_by_pool AS (
```

Comment:
doesn't seem to make sense to me to have this in both locations. It's not necessarily related to an epoch, so maybe make it a pool attribute? It makes the most sense to me in nextEpochStats tho.